Utilize pygame instead of OpenCV for smaller exe size (camera module)

AWeirDKiD opened the pull request:
Rewrote the whole webcam photo command using pygame instead of OpenCV. I made sure to retain the original structure of the program as well as full functionality. 
Only thing to note is that the `time.sleep()` value may have to be increased although the current one (0.5 sec) worked just fine for my testing.

After some rough calculations I believe this change can achieve up to an 76% executable file size decrease from the previous versions.

whois-hoeless commented:
sounds like a W PR to me, I will let mategol and zorro take a look at it and we can merge pretty soon. Thanks for the contribution :)

whois-hoeless commented:
ay even the dev branch works, looking good so far

AWeirDKiD commented:
Implosion will now also delete the registry key set by Pysilon. I took some time to test it and it seems to be working just fine.

AWeirDKiD commented:
- Removed a useless get requests that was doin nothing (along with a loop)
- Added new variable `website_url` as `website` was bein used for 2 tasks
- `cookies.ready` will now be created which for some reason was missing entirely from the old script (wtf)

Should be working as expected but its hella slow due to the method used. I didn't wanna modify it too much so I only made basic changes.

mategol commented:
> Implosion will now also delete the registry key set by Pysilon

But it's already deleting the registry key.
Please remove the part with registry deletion adn then I will merge if this will work flawlessy

AWeirDKiD commented:
> > Implosion will now also delete the registry key set by Pysilon
> 
> But it's already deleting the registry key. Please remove the part with registry deletion adn then I will merge if this will work flawlessy

Sure thing. I didn't notice it.
